<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Hub</class>
 <widget class="QDialog" name="Hub">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>922</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Set background color for the entire page */
QWidget {
    background-color: #ffffff; /* White background */
}

/* Style for the QLabel elements */
QLabel {
    color: #000000; /* Black text */
}

/* Style for the QLineEdit elements */
QLineEdit {
    background-color: #f0f0f0; /* Light gray background */
    border: 1px solid #000000; /* Black border */
    padding: 5px; /* Add some padding */
    color: #000000; /* Black text */
}

/* Style for the QComboBox elements */
QComboBox {
    background-color: #f0f0f0; /* Light gray background */
    border: 1px solid #000000; /* Black border */
    color: #000000; /* Black text */
}

/* Style for the QPushButton elements */
QPushButton {
    background-color: #4CAF50; /* Green background */
    color: #ffffff; /* White text */
    border: 1px solid #4CAF50; /* Green border */
    padding: 5px 10px; /* Add padding */
}

/* Style for the QPushButton when hovered */
QPushButton:hover {
    background-color: #45a049; /* Darker green on hover */
    border: 1px solid #45a049; /* Darker green border on hover */
}

/* Style for the QPushButton when pressed */
QPushButton:pressed {
    background-color: #3d8b40; /* Even darker green on press */
    border: 1px solid #3d8b40; /* Even darker green border on press */
}
</string>
  </property>
  <widget class="QTabWidget" name="Tab">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>891</width>
     <height>621</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/* Set background color for the entire page */
QWidget {
    background-color: #ffffff; /* White background */
}

/* Style for the QLabel elements */
QLabel {
    color: #000000; /* Black text */
}

/* Style for the QLineEdit elements */
QLineEdit {
    background-color: #f0f0f0; /* Light gray background */
    border: 1px solid #000000; /* Black border */
    padding: 5px; /* Add some padding */
    color: #000000; /* Black text */
}

/* Style for the QComboBox elements */
QComboBox {
    background-color: #f0f0f0; /* Light gray background */
    border: 1px solid #000000; /* Black border */
    color: #000000; /* Black text */
}

/* Style for the QPushButton elements */
QPushButton {
    background-color: #4CAF50; /* Green background */
    color: #ffffff; /* White text */
    border: 1px solid #4CAF50; /* Green border */
    padding: 5px 10px; /* Add padding */
}

/* Style for the QPushButton when hovered */
QPushButton:hover {
    background-color: #45a049; /* Darker green on hover */
    border: 1px solid #45a049; /* Darker green border on hover */
}

/* Style for the QPushButton when pressed */
QPushButton:pressed {
    background-color: #3d8b40; /* Even darker green on press */
    border: 1px solid #3d8b40; /* Even darker green border on press */
}
</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="EasyFix">
    <attribute name="title">
     <string>EasyFix</string>
    </attribute>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>881</width>
       <height>551</height>
      </rect>
     </property>
     <widget class="QLabel" name="label2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;Math Sans Serif&quot;;</string>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_notif">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>70</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Envoi requÃªte</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_idp">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>40</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    border: 2px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    background: white;
    selection-background-color: darkgray;
}</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;Math Sans Serif&quot;;</string>
      </property>
      <property name="text">
       <string>PANNE</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_statusp">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>110</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    border: 2px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    background: white;
    selection-background-color: darkgray;
}</string>
      </property>
     </widget>
     <widget class="QTableView" name="tab_panne">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>191</y>
        <width>841</width>
        <height>351</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="Historique2">
    <attribute name="title">
     <string>Historique</string>
    </attribute>
    <widget class="QTextBrowser" name="tab_histp">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>250</y>
       <width>881</width>
       <height>301</height>
      </rect>
     </property>
    </widget>
    <widget class="QCalendarWidget" name="calendarWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>881</width>
       <height>241</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
